src/torchjd/autojac/_transform/jac.py

```diff
@@ -22,6 +22,11 @@ def __init__(
         self.chunk_size = chunk_size
 
     def _differentiate(self, jac_outputs: Sequence[Tensor]) -> tuple[Tensor, ...]:
+        """
+        Differentiates the outputs with respect to the inputs, and right-multiplies them with the
+        provided jac_outputs. Returns the obtained tuple of Jacobians.
```

Comment:
Is it left-multiply? We multiply jacobians on the left with preference vectors, right?

If possible, it would be nice to say that the `jac_outputs` are vectors.